src/pages/tags/[tag].tsx

```diff
@@ -70,22 +71,28 @@ const TagPage: NextPage<TagPageProps> = ({ tagId, wikis }: TagPageProps) => {
         }}
       />
       <Box bgColor="pageBg" border="solid 1px transparent" pb={12}>
```

Comment:
```suggestion
      <Box bgColor="pageBg" mt={-2} border="solid 1px transparent" pb={12}>
```

This fixes the gap between the page and navbar

![CleanShot 2022-06-27 at 12 53 42@2x](https://user-images.githubusercontent.com/52039218/175883059-6d331661-d915-4dfc-99e9-01c1b7ca33d6.png)